modules/microshift-cni-multus-using-ipam.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift_multiple_networks/microshift-cni-multus-using.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="IP-address-management-types-and-additional-networks_{context}"]
+= IP address management types and additional networks
+
+IP addresses are provisioned for an additional network through an IP Address Management (IPAM) CNI plugin that you configure. Supported IP address provisioning types in {microshift-short} are `host-local`, `static`, and `dhcp`.
+
+[id="bridge-interface-specifics_{context}"]
+== bridge interface specifics
+When using `bridge` and the `dhcp` IPAM, a DHCP server listening on the bridge interface is required. If you are using a firewall, configuring the firewalld service by running the `firewall-cmd --remove-service=dhcp` command to allow DHCP traffic on the interface or network zone is also required.
+
+[id="macvlan-interface-specifics_{context}"]
+== macvlan interface specifics
+The `macvlan` type interface accesses the network that the host is connected to. This means that the interface receives and IP address from the libvirt network DHCP server, for example, an IP address from the range of `192.168.122.0/24`.
```

Comment:
I know this is WIP but just did a drive by and want to mention that use of libvirt and the DHCP services it provides is very specific to a deployment of MicroShift that might be done on a VM. A production environment, where MicroShift is installed on an edge device or other dedicated hardware will likely not be using these. Maybe you could say something like, "This means the interface will be assigned an IP address from the host's subnet".  However, I'll defer to a networking SME, for something more appropriate, as I'm not certain dhcp is even required to be used; I would guess other means of static assigned are also possible?

Reply:
+1, suggest to change to ".. receives an IP address from the DHCP server on the host network", and remove the ", for example ....." rest of the sentence.